### PR TITLE
PE-1298: missing translation for conflicting folders

### PR DESCRIPTION
**Pull request by matibat**

### CODE REVIEW

File: lib/l10n/app_es.arb [hunk 1]
```diff
@@ -815,6 +815,21 @@
       }
     }
   },
+  "duplicateFolders": "Carpetas con conflictos",
```
Comment (agsuy) [inline lib/l10n/app_es.arb:818]: "Se encontro una carpeta duplicada"

Comment (matibat) [inline lib/l10n/app_es.arb:818]:
```suggestion
  "duplicateFolders": "El nombre de la carpeta ya existe",
```

File: lib/l10n/app_en.arb [hunk 2]
```diff
@@ -793,7 +793,7 @@
   "@login": {
     "description": "The login process"
   },
-  "duplicateFiles": "{numberOfDuplicateFiles,plural, =0{No duplicate files found} =1{A duplicate file found} =other{{numberOfDuplicateFiles} duplicate files found}}",
```
Comment (matibat) [inline lib/l10n/app_en.arb:796]: We don't ever report Zero duplicate entitites

File: lib/l10n/app_en.arb [hunk 3]
```diff
@@ -815,9 +815,16 @@
       }
     }
   },
-  "duplicateFolders": "Folder Name Conflict",
+  "duplicateFolders": "{numberOfDuplicateFolders,plural, =1{A duplicate folder found} =other{{numberOfDuplicateFolders} duplicate folders found}}",
```
Comment (matibat) [inline lib/l10n/app_en.arb:818]: This was missing a plural to be in pair with `duplicateFiles`

**Review by karlprieb — APPROVED** [review]

File: lib/l10n/app_en.arb [hunk 4]
```diff
@@ -793,7 +793,7 @@
   "@login": {
     "description": "The login process"
   },
-  "duplicateFiles": "{numberOfDuplicateFiles,plural, =0{No duplicate files found} =1{A duplicate file found} =other{{numberOfDuplicateFiles} duplicate files found}}",
+  "duplicateFiles": "{numberOfDuplicateFiles,plural, =1{A duplicate file found} =other{{numberOfDuplicateFiles} duplicate files found}}",
```
Comment (matibat) [inline lib/l10n/app_en.arb:796]:
```suggestion
  "duplicateFiles": "{numberOfDuplicateFiles,plural, =1{A duplicate file found} =other{Duplicate files found}}",
```

**Pull request merged**